### PR TITLE
Rename container class

### PR DESCRIPTION
**Pull request by dallasbpeters**
In this PR we have
1. renamed the container class that was seen in several files
2. renamed the build scripts
3. removed the old eb-scripture-styles.css
4. built new api-bible-content.css

### CODE REVIEW

File: scss/_eb-reset.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // For now the following seems to get the job done. However we may determine that something more agressive like http://cleanslatecss.com might need consideration.
```
Comment (bryceallison) [inline scss/_eb-reset.scss]: @dallasbpeters can you also rename this file?

File: scss/api-bible-content.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@charset "UTF-8";
+
+@import "settings";
+@import "eb-reset";
```
Comment (bryceallison) [inline scss/api-bible-content.scss]: @dallasbpeters let’s rename this too

File: scss/_eb-reset.scss (hunk 1 above)
Comment (dallasbpeters) [reply scss/_eb-reset.scss]: done!

File: scss/api-bible-content.scss (hunk 2 above)
Comment (dallasbpeters) [reply scss/api-bible-content.scss]: done!

**Comment by bryceallison** [comment]
@dallasbpeters almost done… update `Passage.js` and rerun `npm run build`

**Pull request merged**